app/views/distributions/tumbleweed.html.erb

```diff
@@ -124,6 +125,38 @@
           <%= _('These ports are not officially supported, and they are not as stable as official distributions.') %>
         </div>
 
+        <h3 class="my-3">aarch64</h3>
+        <div class="card-deck">
+          <div class="card">
+            <div class="card-body">
+              <h4 class="card-title">
+                <span class="typcn typcn-download-outline"></span>
+                <%= _("DVD Image") %> (4.7GB)
+              </h4>
+              <h6 class="card-subtitle mb-2 text-muted"><%= _("For DVD and USB stick") %></h6>
+              <p class="card-text text-muted"><%= _("Contains a large collection of software for desktop or server use. Suitable for installation or upgrade.") %></p>
+              <a href="https://download.opensuse.org/ports/aarch64/tumbleweed/iso/openSUSE-Tumbleweed-DVD-aarch64-Current.iso" class="card-link"><%= _('Direct Link') %></a>
+              <a href="https://download.opensuse.org/ports/aarch64/tumbleweed/iso/openSUSE-Tumbleweed-DVD-aarch64-Current.iso.meta4" class="card-link"><%= _('Metalink') %></a>
+              <a href="https://download.opensuse.org/ports/aarch64/tumbleweed/iso/openSUSE-Tumbleweed-DVD-aarch64-Current.iso?mirrorlist" class="card-link"><%= _('Pick Mirror') %></a>
+<!-- Currently sha256 file is missing <a href="https://download.opensuse.org/ports/aarch64/tumbleweed/iso/openSUSE-Tumbleweed-DVD-aarch64-Current.iso.sha256" class="card-link"><%= _('Checksum') %></a> -->
```

Comment:
why is there not checksum? :confused:

Reply:
Probably a bug in the publish script. A bugzilla report is opened here: https://bugzilla.opensuse.org/show_bug.cgi?id=1098215